src/game-loop/CMakeLists.txt

```diff
@@ -366,4 +371,5 @@ target_link_libraries(GameLoop
         Audio
         glad
         GraphicsUtils
+        imgui
```

Comment:
Add `SPELUNKY_PSP_WITH_IMGUI` CMake variable + set it conditionally depending on `SPELUNKY_PSP_PLATFORM_DESKTOP` present.

Reply:
...and link `imgui` conditionally, depending on `SPELUNKY_PSP_WITH_IMGUI` being present.